backends/s3.go

```diff
@@ -0,0 +1,178 @@
+package backends
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+
+	"github.com/bndw/pick/errors"
+)
+
+const (
+	defaultS3Bucket = "pick"
+	defaultS3Key    = "safes/default.safe"
+	// Backups
+	defaultS3BackupTimeFormat  = "2006-01-02_15-04-05"
+	defaultS3BackupKeyTemplate = "backups/%s.safe"
+)
+
+// S3Backend provides AWS S3 backed safe storage
```

Comment:
Agreed, documentation in general is lacking for this project.

I've been working in that area and will update soon :)